app/actions/CommonActions.scala

```diff
@@ -12,7 +15,7 @@ object CommonActions {
 
   val GoogleAuthAction = OAuthActions.AuthAction
 
-  val GoogleAuthenticatedStaffAction = NoCacheAction andThen GoogleAuthAction
+  val GoogleAuthenticatedStaffAction = NoCacheAction andThen ProdOnlyAuth
```

Comment:
This replaces and wraps the old google auth action, which in turn was just calling `OAuthActions.AuthAction`